modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/PersistentPageMemoryMvPartitionStorage.java

```diff
@@ -212,13 +212,30 @@ public void lastApplied(long lastAppliedIndex, long lastAppliedTerm) throws Stor
     }
 
     private void lastAppliedBusy(long lastAppliedIndex, long lastAppliedTerm) throws StorageException {
+        updateMeta((lastCheckpointId, meta) -> meta.lastApplied(lastCheckpointId, lastAppliedIndex, lastAppliedTerm));
+    }
+
+    /**
+     * Closure interface for {@link #update(UUID, PartitionMeta)}.
+     */
```

Comment:
```suggestion
    /** Closure interface for {@link #update(UUID, PartitionMeta)}. */
```

Reply:
There's no rule to force comments into a single line if they fit. And, as far as I see in the project, such comments are usually attributed to fields, not classes or methods. Again, what's your motivation?